core/script_language.h

```diff
@@ -200,6 +200,34 @@ class ScriptInstance {
 	virtual ~ScriptInstance();
 };
 
+struct ScriptCodeCompletionOption {
+	enum Kind {
```

Comment:
As for you other PR, "Kind" is not the correct word here but "Type".

Reply:
I'm not really good at English.
I just followed the LSP `CompletionItem` https://code.visualstudio.com/api/references/vscode-api#CompletionItem